packages/fungible-token/bridge-fungible-token/fuel-toolchain.toml

```diff
@@ -2,5 +2,5 @@
 channel = "latest-2023-07-20"
 
 [components]
-forc = "0.42.1"
-fuel-core = "0.19.1"
+forc = "0.44.0"
+fuel-core = "0.20.3"
```

Comment:
```suggestion
fuel-core = "0.20.4"
```

Reply:
If I change it to 0.20.4 I got an error `component 'fuel-core' not found in currently active toolchain 'latest-2023-07-20-aarch64-apple-darwin'`. This is probably because the 0.20.4 is in another channel but I didn't find one that works.

Reply:
ok no worries